vector/v.to.db/v.to.db.html

```diff
@@ -168,8 +168,3 @@ <h2>AUTHOR</h2>
 
 Radim Blazek, ITC-irst, Trento, Italy<br>
 Line sinuousity implemented by Wolf Bergenheim
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
`<h2>REFERENCES</h2>` section should be before `<h2>SEE ALSO</h2>`.

`<h2>AUTHOR</h2>` should be `<h2>AUTHORS</h2>`.